apps/explorer/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -190,16 +192,17 @@ export function TransactionView({
     const isSponsoredTransaction = gasOwner !== sender;
 
     const timestamp = transaction.timestampMs;
+    const transactionKindName = getTransactionKindName(
+        getTransactionKind(transaction)!
+    );
 
     return (
         <div className={clsx(styles.txdetailsbg)}>
             <div className="mt-5 mb-10">
                 <PageHeader
                     type="Transaction"
                     title={getTransactionDigest(transaction)}
-                    subtitle={getTransactionKindName(
-                        getTransactionKind(transaction)!
-                    )}
+                    subtitle={transactionKindName}
```

Comment:
Can we tweak this so that this only displays for non-programmable transactions?

Reply:
fixed